src/components/AllMovies/AllMovies.js

```diff
@@ -1,26 +1,61 @@
-import React from 'react'
+import React, { Component } from 'react'
 import Movie from '../Movie/Movie'
 import './AllMovies.css'
+import { fetchAllMovies } from '../../utilities/ApiCalls'
 
-const AllMovies = ({ movieData }) => {
+class AllMovies extends Component {
```

Comment:
Excited about switching where the state for the movie data lives--this flows much nicer!